collections/_article/progress-report-state-of-particles.md

```diff
@@ -0,0 +1,94 @@
+
+-----
+
+Godot 4 offers a great opportunity to make some long-requested and much needed changes in many areas, and one of those areas is particle nodes and resources. Before 4.0 some settings used to be a bit confusing and clunky to work with. Now a lot of things have changed: parameters are set with a range using min and max boundaries instead of value and random deviation; there's a particle collision system; there's a turbulence system, there are sub-emitters, and much more.
+
+All around, particles already received a lot of love during the development of Godot 4. However, a number of key features were still missing as of 4.1. Below I go into detail about additions and improvements made to the particle system in the last few months.
+
+<video autoplay loop muted playsinline title="A countdown with particle effects">
+  <source src="/storage/blog/state-of-particles/final-countdown.mp4" type="video/mp4">
+</video>
+
+## Artistic control
+
+Godot's existing particle system made an assumption that people would use it for more physically accurate simulations, such which are suited for more realistic games. But in practice a vast majority of projects in the Godot ecosystem choose a stylized art direction.
+
+This creates a big pain point when using current particles. In order to obtain an exaggerated motion and snappy timings in visual effects, acceleration-based approach just doesn't cut it. It ends up really clunky, hard to use, and sometimes achieving certain effects is just not possible.
+
+To address these issues, and more, the `GPUParticles` system has been refactored and expanded upon. Particles can now support not just realistic types of effects, but also more stylized ones with more exaggerated motion. Additionally, a couple of long-requested quality of life features was added.
+
+## The new `GPUParticles`
+
+**NOTE:** These changes should be 100% backwards compatible. If particles look different after updating, please [file a bug report](https://github.com/godotengine/godot/issues/new/choose).
+
+_[Pioneering of this work](https://github.com/godotengine/godot/pull/78851) was kindly sponsored by [PahdoLabs](https://www.pahdolabs.com/)._
+
+### New parameters for `GPUParticles` nodes
+
+Two new parameters were added to `GPUParticles2D`/`GPUParticles3D` nodes:
+
+- `amount_ratio`: from 0 to 1, a percentage of particles compared to the amount that will be emitted.
+- `interp_to_end`: from 0 to 1, allows to interpolate all particles from the same node simultaneously towards the end of their lifetime.
+
+### Changes in `ParticleProcessMaterial`
+
+The main change in the particle material is the big-big refactor of the internal code. The code was split into smaller, reusable functions, which are
+called both from `start()` and `process()`. This ensures better consistency and more maintainability.
+
+Multiple configuration options were added or changed to expand your control over damping, emission, and velocity.
```

Comment:
```suggestion
Multiple configuration options were added or changed to expand control over damping, emission, and velocity.
```

Reply:
I mean, we're talking to the reader 🙃 But fine